pages/dmp_supporting_info/support_00_rdm_terms.md

```diff
@@ -0,0 +1,63 @@
+Preservation: An activity within archiving in which specific items of data are maintained over time so that they can 
+still be accessed and understood through changes in technology. 
+
+### Data storage
+
+### FAIR principles
+
+### Information security
+
+### Metadata
+#### [CODATA RDM Terminology (version 2023)](https://doi.org/10.5281/zenodo.10626170) [![License: CC BY 4.0](https://img.shields.io/badge/License-CC_BY_4.0-lightgrey.svg)](https://creativecommons.org/licenses/by/4.0/)
+Metadata: Data about data. It is data (or information) that defines and describes the characteristics of other data. It is used to improve the understanding and use of the data.
+
+### Metadata standard
+#### [CODATA RDM Terminology (version 2023)](https://doi.org/10.5281/zenodo.10626170) [![License: CC BY 4.0](https://img.shields.io/badge/License-CC_BY_4.0-lightgrey.svg)](https://creativecommons.org/licenses/by/4.0/)
+Metadata standard: High level, shared representation of the metadata elements related to a dataset, collection, or other digital object. May also provide an XML schema describing the format in which the elements should be stored. Typically, a standard XML format is defined using XML Schema or document type definition (DTD). Standards are typically ratified by national or international standards bodies.
+
+### Persistand identifier (PID)
+#### [CODATA RDM Terminology (version 2023)](https://doi.org/10.5281/zenodo.10626170) [![License: CC BY 4.0](https://img.shields.io/badge/License-CC_BY_4.0-lightgrey.svg)](https://creativecommons.org/licenses/by/4.0/)
+Persistent identifier: Long-lasting digital reference to an object that gives information about that object regardless of what happens to that object. Developed to address link rot, a persistent identifier can be resolved to provide an appropriate representation of an object whether that object changes its online location or goes offline.
+
+Examples:
+* [Digital Object identifier (DOI)](https://en.wikipedia.org/wiki/Digital_object_identifier "2024-09-09")
+* [Handle](https://en.wikipedia.org/wiki/Handle_System "2024-09-09")
+* Persistent URL (PURL)
+* Compact identifiers for life sciece data resolving via [identifers.org](https://identifiers.org/)
+
+### Personal data
+
+### Provencance, data lineage
+#### [CODATA RDM Terminology (version 2023)](https://doi.org/10.5281/zenodo.10626170) [![License: CC BY 4.0](https://img.shields.io/badge/License-CC_BY_4.0-lightgrey.svg)](https://creativecommons.org/licenses/by/4.0/)
+Provenance: A type of historical information or metadata about the origin, location or the source of something, or the history of the ownership or location of an object or resource including digital objects. For example, information about the Principal Investigator who recorded the data, and the information concerning its storage, handling, and migration.
+
+### Research data
+
+### Sensitive data
```

Comment:
```suggestion
### Sensitive data
#### [RDMkit: Data sensitivity](https://rdmkit.elixir-europe.org/data_sensitivity)

>  The term sensitive data is used when making data publicly available could put people, organisations, countries, and/or ecosystems at risk - this could be for example, personal or commercial information, [...]. Such data sensitivity must be protected against unauthorized access, and therefore one should be cautious when deadling with potentitally sensitive or sensitive information.

### Special category data
#### [European Commission: What personal data is considered sensitive?](https://commission.europa.eu/law/law-topic/data-protection/reform/rules-business-and-organisations/legal-grounds-processing-data/sensitive-data/what-personal-data-considered-sensitive_en)
The GDPR defines special categories of personal data (‘[personal] sensitive data’) as:
> -   personal data revealing racial or ethnic origin, political opinions, religious or philosophical beliefs;
> - trade-union membership;
> -  genetic data, biometric data processed solely to identify a human being;
> - health-related data;
> - data concerning a person’s sex life or sexual orientation.


```